lib/trader.js

```diff
@@ -64,8 +64,17 @@ Trader.prototype.syncLogs = function syncLogs () {
     .then(data => data.body)
   // Delete log files that are two or more days old
     .then(() => {
-      let today = new Date()
-      return logs.removeLogFiles(today.setDate(today.getDate() - 2))
+      const twoDaysAgo = (() => {
+        let date = new Date()
+
+	// Notice that `setDate()` can take negative values. So if you'd take
+	// -2 days on the 1st of April you'd get the 30th of March. Several
+	// examples can be seen at: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate#Using_setDate()
+        date.setDate(date.getDate() - 2)
+	return date
+      })()
+
+      return logs.removeLogFiles(twoDaysAgo)
```

Comment:
now return the param, that way this function runs without interfering with queryNewestLogs

```
return it
```